packages/popmotion-pose/src/index.ts

```diff
@@ -1,3 +1,5 @@
 import domPose from './dom/pose';
+import { setReducedMotion } from './inc/accessibility';
```

Comment:
u can just do:
```js
export { default } from './dom/pose';
export { setReducedMotion } from './inc/accessibility';
```

Reply:
Ah that's nice thanks though I've taken it out for now in favour of forcing animations in 'development'

Reply:
I'd rather see this as an option to leave it up to a developer to choose when to disable/enable the animation

e.g. somewhere in a config set `disableAnimation: true` and have popmotion respect that rather than dealing with prefers reduce motion and browser inconsistencies between it, the code written in this library to support is will probably be similarly written within the app the user has written as well which just makes more code bloat in the resulting app.